crates/config/src/fuzz.rs

```diff
@@ -51,8 +51,9 @@ impl Default for FuzzConfig {
 impl FuzzConfig {
     /// Creates fuzz configuration to write failures in `{PROJECT_ROOT}/cache/fuzz` dir.
     pub fn new(cache_dir: PathBuf) -> Self {
+        let cache_dir = cache_dir.display();
         Self {
-            failure_persist_dir: Some(cache_dir),
+            failure_persist_dir: Some(format!("{cache_dir}/fuzz").into()),
```

Comment:
use cache_dir.join()

Reply:
yeah, I reverted the changed for failure persisted dir as they're not quite needed and corpus can now be defined as relative or absolute (external from project) path https://github.com/foundry-rs/foundry/pull/10190/commits/d305fffa516d05a2f790d7a8272618d442678722